backend/config.js

```diff
@@ -173,38 +173,38 @@ var environments = {
                 lastName: 'Test',
                 roleID: 1,
                 scopes: 'admin',
-                password: 'testuser12345',
+                password: 'Testadmin12345',
             },
             admin: {
-                email: 'test-adm@mail.net',
-                firstName: 'Admin',
-                lastName: 'Test',
+                email: process.env.ADMIN_USER_EMAIL,
+                firstName: process.env.ADMIN_USER_FIRST_NAME,
+                lastName: process.env.ADMIN_USER_LAST_NAME,
                 roleID: 2,
-                password: 'testadmin12345',
+                password: 'Adminuser1',
```

Comment:
I kept this password the same as the one that I used to create kate's admin account on staging.